Update README.md (#320)
jfrog/jfrog-vscode-extension: README.md

By default, paths containing the words `test`, `venv` and `node_modules` are excluded from Xray scan.
The exclude pattern can be configured in the [Extension Settings](#extension-settings).

> ### Exclude Development Dependencies During Scan

Reviewer: Please add it to the table of contents

> Development dependencies are scanned by default. You can skip it by choosing `Exclude Dev Dependencies` under Settings.

Reviewer: ```suggestion
Development dependencies are scanned by default. You can exclude them by choosing `Exclude Dev Dependencies` in the [Extension Settings](#extension-settings).
```

By default, paths containing the words `test`, `venv` and `node_modules` are excluded from Xray scan.
The exclude pattern can be configured in the [Extension Settings](#extension-settings).

> ### Exclude development dependencies during the scan

Reviewer 2: I think we should also add it in the table of content, what do you think?
